dev-games/osgQt/osgQt-3.5.5.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils vcs-snapshot
```

Comment:
Do you actually need vcs-snapshot here? I think github gets the directory name right when tag is named after version.